infrastructure/template/plugin/.editorconfig

```diff
@@ -9,6 +9,9 @@ insert_final_newline = true
 [*.py]
 indent_style = space
 indent_size = 4
+# Black compatible values for isort https://black.readthedocs.io/en/stable/compatible_configs.html#isort
+profile = black
```

Comment:
These should probably be in setup.cfg [isort] block?

Reply:
Unfortunately isort ignores setup.cfg if .editorconfig is present.

Reply:
Could you verify that again? According to https://pycqa.github.io/isort/docs/configuration/config_files/ and to my experience setup.cfg supersedes .editorconfig which is the last hope for isort.

Reply:
Yeah, I realized that this was caused by .editorconfig lying in the deeper lever in the file structure. I moved it next to the setup.cfg and it worked right away! :)